pkg/common/args_test.go

```diff
@@ -44,7 +47,9 @@ func TestValidateArgs(t *testing.T) {
 			skip:          "",
 			expectedSkip:  "",
 			extraArgs:     []string{},
-			expectedArgs:  "",
+			expectedArgs:  []string{},
```

Comment:
@rakshitgondwal expectedArgs is a  a string and not slice of string since that is what the container expects.